args/physics/HalfEdgeEdge.h

```diff
@@ -26,6 +26,16 @@ namespace args::physics
 			nextEdge = newNextEdge;
 			prevEdge = newPrevEdge;
 		}
+
+		math::vec3 getLocalNormal() const
+		{
+			return face->normal;
+		}
+
+		math::vec3 getLocalEdgeDirection()  const
+		{
+			return *nextEdge->edgePositionPtr - *edgePositionPtr;
+		}
```

Comment:
These should receive documentation, also the filename should be `halfedgeedge.hpp` or simmilar